roles/kubernetes/preinstall/templates/openstack-cloud-config.j2

```diff
@@ -8,9 +8,15 @@ tenant-id={{ openstack_tenant_id }}
 domain-name={{ openstack_domain_name }}
 {% endif %}
 
+{% if openstack_blockstorage_version is defined %}
+[BlockStorage]
+bs-version={{ openstack_blockstorage_version }}
+{% endif %}
+
 {% if openstack_lbaas_enabled and openstack_lbaas_subnet_id %}
 [LoadBalancer]
 subnet-id={{ openstack_lbaas_subnet_id }}
+floating-network-id={{ openstack_lbaas_floating_network_id }}
```

Comment:
This needs a check if it's defined because you document it as being disabled by default.

Reply:
```
{% if openstack_lbaas_floating_network_id is defined %} 
floating-network-id={{ openstack_lbaas_floating_network_id }}
{% endif %}
```